.github/workflows/test-hyperlinks.yml

```diff
@@ -146,7 +146,7 @@ jobs:
               echo $FILE >> missingFiles.txt
               continue
             fi  
-            grep -onHE -e "\]\([^\)]+" -e "\`\`\`" -e "http://[^\ \;\"\'\<\>\,\`\)]+" -e "https://[^\ \;\"\'\<\>\,\`\)]+" ${FILE} | sed 's/](//'  > links.tmp
+            grep -onHE -e "\]\([^\)]+" -e "\`\`\`" -e "\(http://[^\ \;\"\'\<\>\,\`\)]+" -e "\(https://[^\ \;\"\'\<\>\,\`\)]+" ${FILE}  | sed 's/](//'  > links.tmp
```

Comment:
The two http & https -e checks could be combined by doing

`-e "\(https?://[^\ \;\"\'\<\>\,\`\)]+"`

Just make the s an optional match. But are we sure the leading and following \( & \) are necessary?